backend/docs/api/management_v2_deployments.yaml

```diff
@@ -0,0 +1,1286 @@
+openapi: 3.0.1
+info:
+  description: |
+    Version 2 of the API for deployments management.
+    Intended for use by the web UI.
+  title: Deployments v2
+  version: "2"
+servers:
+- url: https://hosted.mender.io/api/management/v2/deployments
+paths:
+  /deployments:
+    get:
+      description: |
+        Returns a filtered collection of deployments in the system. While you can provide multiple
+        deplyoment identifiers, and multiple names to get the multiple deplyoments you cannot mix
+        the ids and names in a one query expecting to get deployments that match both names or ids.
+        The endpoint will return the deployments that match the ids and the name, if combined.
+      operationId: List Deployments
+      parameters:
+      - description: |
+          Deployment identifier. You can provide it multiple times to query a set of deployments.
+        in: query
+        name: id
+        schema:
+          type: string
+      - description: |
+          Deployment name. You can provide it multiple times to query a set of deployments.
+        in: query
+        name: name
+        schema:
+          type: string
+      - description: Deployment status filter.
+        in: query
+        name: status
+        schema:
+          enum:
+          - inprogress
+          - finished
+          - pending
+          type: string
+      - description: |
```

Comment:
it seems to be that we use `|` randomly, here we have it, but on the line 50 below with ` - description: Results page number`  we dont.